tests/client/amo/components/TestAddonDetail.js

```diff
@@ -10,7 +10,7 @@ import AddonDetail, { allowedDescriptionTags }
 import I18nProvider from 'core/i18n/Provider';
 import InstallButton from 'core/components/InstallButton';
 
-import { getFakeI18nInst } from 'tests/client/helpers';
+import { getFakeI18nInst } from '../../helpers';
```

Comment:
We can add the project root but it feels a bit risky to me. If you look at our project root we have a lot of directories. For example, `./config` is a directory but we also use the `config` from node_module as well. This actually confuses the linter in some places (which may be a bug). It doesn't cause errors though so we could do it.

The other alternative is to push tests into another subdirectory like `tests/testlib` and import everything from `testlib`.